### ISSUE TITLE
Security Exception while resolving plugin

### ISSUE THREAD

**Issue by manish7-thakur** [scalapb/ScalaPB#860]
![Screen Shot 2563-06-22 at 11 49 17](https://user-images.githubusercontent.com/3353463/85249613-7335b500-b47e-11ea-8382-0efc733179f2.png)

Just recently got certificate exception while resolving the sbt scala plugin.

scala version: 2.11 & 2.12

protoc version
```
addSbtPlugin("com.thesamet" % "sbt-protoc" % "0.99.27")

libraryDependencies += "com.thesamet.scalapb" %% "compilerplugin" % "0.9.6"
```

**Comment by thesamet** [comment]
Hi @manish7-thakur , this issue seems like a local configuration issue of your system (old JDK maybe?) or some intermittent networking issue. Based on the information you provided, it's hard to help further.

**Comment by manish7-thakur** [comment]
@thesamet It's been there for couple of days, I'll see to it. Meanwhile is there any other info you might need that can help us track it down ?

**Comment by manish7-thakur** [comment]
@thesamet It got solved by updating sbt to version 1.0.XX, I'll close this
